software_training_assignment/src/publisher.h

```diff
@@ -0,0 +1,29 @@
+#include <ros/ros.h>
+#include <software_training_assignment/Position.h>
+#include <turtlesim/Pose.h>
+#include <cmath>
+
+namespace P
+{
+    class Publisher
```

Comment:
shouldn't use Publisher as a name, it will be confusing. Instead be more descriptive - PositionPublisher